src/Pages/Home.jsx

```diff
@@ -37,17 +35,16 @@ export default function Home() {
                     </div> : null
                     }
 
-                    {(notes.length !== 0) ?
+                    {(filteredNoteList.length !== 0) ?
```

Comment:
You can use an operator other than ternary operator

Reply:
I'll Look into it. Thank you.